### DISCUSSION TITLE
Can't edit  /etc/network/if-up.d/eQ3StartNetwork to add /etc/hosts entry

### DISCUSSION THREAD

**Discussion by simonbuehler** [OpenCCU/OpenCCU#1691] [🇺🇸 Q&A] [answered]
hi,

in a very resticted environment ther is no dns resolver available so the route to a vhost must be set via a entry in `/etc/hosts` but gets removed on restart.

i can't edit the `/etc/network/if-up.d/eQ3StartNetwork` file to make it permanent (readonly), is there quick fix for this?

**Comment by jens-maus** [comment, marked as answer]
Converted this to a discussion thread instead, since this is no issue or bug. This is indeed the expected behaviour and you are not supposed to manually edit `/etc/hosts` or `/etc/network/if-up.d/eQ3StartNetwork` as this is part of the read-only part of the RaspberryMatic firmware. Thus, either use an IP or install/setup a local dns resolver.
